src/components/newInvoice/NewInvoiceForm.js

```diff
@@ -71,7 +66,11 @@ const NewInvoiceForm = () => {
             const invoice = { ...values, creationDate: saleDate, isPaid, saleDate }
             fetch(`http://localhost:5000/invoices/`, { method: 'POST', headers: { "Content-type": "application/json" }, body: JSON.stringify(invoice) })
                 .then(res => res.json())
-                .then(res => { setAlertMessage('Faktura została dodana', 'pass') })
+                .then(res => {
+                    setAlertMessage('Faktura została dodana', 'pass')
+                    localStorage.clear()
```

Comment:
use localStorage.removeItem() instead. You don't want to clear whole storage, just one specific item.